workloads/maas-gateway-perf/run.sh

```
@@ -0,0 +1,552 @@
+    if oc get ns "$ns" &>/dev/null; then
+      oc delete ns "$ns" --ignore-not-found --timeout=60s 2>/dev/null || {
+        echo "Force-removing finalizers from stuck namespace: $ns"
+        oc get ns "$ns" -o json | jq '.spec.finalizers = []' | oc replace --raw "/api/v1/namespaces/$ns/finalize" -f - 2>/dev/null || true
+      }
+    fi
+  done
+
+  # Remove stale clone
+  rm -rf /tmp/models-as-a-service
+  echo "Cleanup complete"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)